.github/workflows/ci-windows-artifacts.yml

```diff
@@ -56,6 +56,7 @@ jobs:
 
       - name: Prepare libavif (cmake)
         run: >
+          CFLAGS=-static CXXFLAGS=-static LDFLAGS=-static
```

Comment:
IMPORTANT: This should be examined. Does MSVC support the `-static` compiler or linker option? It seems unlikely.

Reply:
I see you are removing this in https://github.com/AOMediaCodec/libavif/pull/2648.